### PR TITLE
FEATURE: [okx] support market trade streaming

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

File: pkg/exchange/okex/parse.go [hunk 1]
```diff
@@ -363,3 +373,59 @@ func parseOrder(v *fastjson.Value) ([]okexapi.OrderDetails, error) {
...
+		return types.SideTypeSell, nil
+
+	default:
+		return types.SideType(side), fmt.Errorf("unexpected side: %s", side)
+	}
+}
+
+type MarketTradeEvent struct {
+	InstId    string                     `json:"instId"`
+	TradeId   string                     `json:"tradeId"`
```
Comment (c9s) [inline pkg/exchange/okex/parse.go:392]: you can use StrInt here?

Comment (c9s) [reply pkg/exchange/okex/parse.go:392]: see pkg/types/strint.go

Comment (c9s) [reply pkg/exchange/okex/parse.go:392]: or maybe you can create a StrUint for this?

**Review by c9s — APPROVED** [review]

File: pkg/exchange/okex/parse.go (hunk 1 above)
Comment (bailantaotao) [reply pkg/exchange/okex/parse.go:392]: I asked for official support, which confirmed that the trade ID will be numeric. Therefore, I made the necessary changes to fix it.

**Review by c9s — APPROVED** [review]

**Pull request merged**